cfme/tests/configure/test_access_control.py

```diff
@@ -51,7 +55,59 @@ def new_role(appliance, name=None):
 
 
 @pytest.fixture(scope='module')
-def child_tenant(appliance, request):
+def two_child_tenants(appliance):
+    child_tenant1 = appliance.collections.tenants.create(
+        name='marketing',
+        description='marketing',
+        parent=appliance.collections.tenants.get_root_tenant()
+    )
+    child_tenant2 = appliance.collections.tenants.create(
+        name='finance',
+        description='finance',
+        parent=appliance.collections.tenants.get_root_tenant()
+    )
+
+    yield child_tenant1, child_tenant2
+    child_tenant1.delete_if_exists()
+    child_tenant2.delete_if_exists()
+
+
+@pytest.fixture
+def setup_openldap_user_group(appliance, two_child_tenants):
+    auth_provider = get_auth_crud('cfme_openldap')
```

Comment:
I don't like the call to `get_auth_crud` and `cfme_openldap` twice, I think this should take the fixture `openldap_auth_provider` as an argument.

Reply:
additionally, I would wrap it in a try/except and catch the `KeyError`

Reply:
^ ignore, it is already doing that

Reply:
@john-dupuy , agreed